backend/api/Controllers/Models/RescheduleMissionQuery.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Api.Database.Models;
+
+namespace Api.Controllers.Models
+{
+    public class RescheduleMissionQuery
+    {
+        public MissionSourceType MissionType { get; set; }
```

Comment:
`MissionSourceType` likely not needed as this is given in the `MissionDefinition`